cmd/ocsp-updater/main_test.go

```diff
@@ -195,9 +195,9 @@ func TestFindStaleOCSPResponses(t *testing.T) {
 	test.AssertNotError(t, err, "Couldn't update ocspLastUpdated")
 
 	earliest := fc.Now().Add(-time.Hour)
-	certs, err := updater.findStaleOCSPResponses(earliest, 10)
-	test.AssertNotError(t, err, "Couldn't find certificate")
-	test.AssertEquals(t, len(certs), 1)
+	statuses, err := updater.findStaleOCSPResponses(earliest, 10)
+	test.AssertNotError(t, err, "Couldn't find status")
+	test.AssertEquals(t, len(statuses), 1)
```

Comment:
If you're changing this naming here, maybe change it in TestFindStaleOCSPResponsesStaleMaxAge too?